app/models/edition/taggable_organisations.rb

```diff
@@ -0,0 +1,21 @@
+module Edition::TaggableOrganisations
+  extend ActiveSupport::Concern
+
+  def can_be_tagged_to_taxonomy?
+    organisations_content_ids = organisations.map(&:content_id)
+
+    has_taggable_organisations?(organisations_content_ids)
```

Comment:
Could we come up with a better name for `taggable_organisations`? The organisations' documents are taggable, not the org itself.